### ISSUE TITLE
Two undefined names

### ISSUE THREAD

**Issue by cclauss** [securestate/king-phisher#228]
flake8 testing of https://github.com/securestate/king-phisher on Python 2.7.13

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./king_phisher/client/dialogs/ssh_host_key.py:187:37: F821 undefined name 'PermissionError'
			except IOError if its.py_v2 else PermissionError:
                                    ^
./tests/client/application.py:50:10: F821 undefined name 'Exceptiongit'
		except Exceptiongit:
         ^
```

**Comment by zeroSteiner** [comment]
The PermissionError can be ignored. It's defined in Python3 and not Python 2.7 which we don't support anymore.

The other looks like a typo and we'll get it fixed soon.

**Comment by zeroSteiner** [comment]
Fixed in commit 68c94bc65596bf39ada84a6aa67a4e4a792d1aba. If you run tools like this again in the future please remember that we no longer support Python 2.7 and you'll need to use the Python3 version.
